lib/nc_time_axis/__init__.py

```diff
@@ -211,3 +198,8 @@ def convert(cls, value, unit, axis):
             raise ValueError('A "calendar" attribute must be attached to '
                              'netcdftime object to understand them properly.')
         return netcdftime.date2num(value, cls.standard_unit, calendar=first_value.calendar)
+
+
+# Register NetCDFTimeConverter with matplotlib.unit's converter dictionary. 
+if netcdftime.datetime not in munits.registry:
+    munits.registry[netcdftime.datetime] = NetCDFTimeConverter()
```

Comment:
@lbdreyer Seems totally fine to me. 

You could say `Automatically register ...` but, really, I'm just splitting hairs here.